api/api.proto

```diff
@@ -3454,6 +3531,110 @@ message SdkRoleUpdateResponse {
   SdkRole role = 1;
 }
 
+// SdkFilesystemTrimStartRequest defines a request to start a background filesystem trim operation
+message SdkFilesystemTrimStartRequest {
+  // Id of the volume
+  string volume_id = 1;
+  // Path where the volume is mounted
+  string mount_path = 2;
+}
+
+// SdkFilesystemTrimStartResponse defines the response for a SdkFilesystemTrimStartRequest. If case of error, this response struct is not valid
```

Comment:
I think you can remove all the `If case of error...` documentation for all of these messages since these are not RPCs, just messages.